packages/bpk-component-switch/src/BpkSwitch.module.scss

```diff
@@ -56,6 +56,11 @@ $border-width: $bpk-one-pixel-rem * 2;
   &__label {
     margin-right: bpk-spacing-md();
     line-height: $height;
+
+    @include bpk-rtl {
+      margin-right: 0;
```

Comment:
Could we use `margin-inline-start` here?

cc: @LexSwed

Reply:
(We haven't decided on its usage in Backpack, but in main web are moving away from bpk-rtl to towards native support where possible)

Reply:
I think this would be safe to use here yes!

Reply:
Make sense of that, thank you both.